uui-components/src/pickers/PickerInputBase.tsx

```diff
@@ -315,7 +315,8 @@ export abstract class PickerInputBase<TItem, TId, TProps> extends PickerBase<TIt
         });
     }
 
-    handleBlur = (e: React.FocusEvent<HTMLElement>) => {
+    handleClickOutside = (e: React.FocusEvent<HTMLElement>) => {
```

Comment:
I suppose that this function will be just a handler for body closing, not an event handler related to any specific case like click outside or blur, so let's rename it to 'closePickerBody', remove from here `this.props.onBlur(e)` and `e` parameter and provide this blur handler directly via `getTogglerProps`.